### PR TITLE
Added Nicer Gallery

### PR DESCRIPTION
**Pull request by razziedazzel**
Added a Gallery that looks more professional.

2021 Gallery linked to Google Photos for host

Adjusted the drop down top menu to acknowledge it being wider.

Removed "More" on the main page as it did not point to anything

### CODE REVIEW

File: gallery/2021/candid/candid.pug [hunk 1]
```diff
@@ -0,0 +1,150 @@
+include mixins/active-link
```
Comment (rkomiyama) [inline gallery/2021/candid/candid.pug:1]: If you use `extends /base`, you won't have to rewrite the same code over and over.

Plus, it keeps the pug files more modular that way so we don't have to, for example, edit the navbar in every single pug file.

File: gallery/2021/other/other.pug [hunk 2]
```diff
@@ -0,0 +1,150 @@
+include mixins/active-link
```
Comment (rkomiyama) [inline gallery/2021/other/other.pug:1]: If you use extends /base, you won't have to rewrite the same code over and over.

File: gallery/2021/portrait/portrait.pug [hunk 3]
```diff
@@ -0,0 +1,150 @@
+include mixins/active-link
```
Comment (rkomiyama) [inline gallery/2021/portrait/portrait.pug:1]: If you use extends /base, you won't have to rewrite the same code over and over.

**Review by rkomiyama — COMMENTED** [review]
Left a few comments.

**Comment by razziedazzel** [comment]
Redid Pug Files to use extends /base

**Comment by razziedazzel** [comment]
added index pug to reflect changes

**Comment by FaelanGalanti** [comment]
@razziedazzel, I noticed a compilation error. It looks like there's a block missing from the `.pug` templates. Should be

```pug
extends /base

block content

    <content>
```

The content block gets pulled into the main body of the base template and wrapped in a `.container-fluid`.

**Pull request closed without merging**